Library/Homebrew/cask/lib/hbc/installer.rb

```diff
@@ -307,7 +328,7 @@ def save_caskfile
     end
 
     def uninstall
-      odebug "Hbc::Installer#uninstall"
+      oh1 "Uninstalling Cask #{@cask}"
```

Comment:
This is intentional so that the parts of the reinstall process that are uninstall parts are clearly delineated from the install parts. `oh1` groups these parts more obviously.